markdown/org/docs/patterns/diana/instructions/en.md

```diff
@@ -1,63 +1,67 @@
-and topstitch along the folded edge to keep it in place.
+- With *good sides together* align the non-folded edge of the strip with the back neckline.
+- Sew the strip to the back neckline and trim the seam. 
+- Turn the strip to the wrong side of the fabric and *topstitch* along the folded edge to keep it in place.
 
-## Close shoulder seams
+### Step 2: Close shoulder seams
 
 ![This drawing was too nice not to use](neckline.jpg)
 
-Finish the raw edge on the front neckline in a way that suits your fabric 
-(if it doesn’t fray, you can leave it unfinished). 
+- Finish the raw edge on the front neckline in a way that suits your fabric (if it doesn’t fray, you can leave it unfinished).
+- With *good sides together*, place the front on the back along the shoulder seams.
+- Fold the front neckline seam allowance over to the wrong side of the back.
 
-Place the front on the back with good sides together, aligning the shoulder seams. 
-Fold the front neckline seam allowance over to the wrong side of the back. 
+<Note>
 
-If you have a lightweight woven fabric you can fold this edge up again, so it will form a narrow
-double hem afterwards (it can help to press these folds first). 
+If you have a lightweight woven fabric you can fold this edge up again, so it will form a narrow double hem afterwards (it can help to press these folds first).
 
-Stitch the shoulder seam, enclosing the raw edge of the back neckline in the fold. 
+</Note>
 
-Turn to the right side and press.
+- Stitch the shoulder seam, enclosing the raw edge of the back neckline in the fold.
+- Turn to the right side and press.
 
-## Finish front neckline
+### Step 3: Finish front neckline
 
-Press the hem formed in your fabric by the folds you made at the shoulder seam when you closed the shoulder and topstitch close to the edge.
+- Press the hem formed in your fabric by the folds you made at the shoulder seam when you closed the shoulder and topstitch close to the edge.
  
-## Attach sleeves
+### Step 4: Attach sleeves
 
-The sleeve will be inserted flat, meaning the armhole seam will be sewn first and the sleeve
-and side seam will be closed in one go. 
+The sleeve will be inserted flat, meaning the armhole seam will be sewn first and the sleeve and side seam will be closed in one go.
 
-Place the sleeve head along the armhole, with *good sides together* and matching notches. 
-If necessary, ease in the sleeve head at the top, between the notches. 
+- With *good sides together*, matching notches, pin the sleeve head along the armhole.
+- If necessary, ease in the sleeve head at the top, between the notches.
+- Sew, finish and press the seam.
+- Repeat for other sleeve.
 
-Sew, finish and press the seam.
+### Step 5: Close side and sleeve seam
 
-## Close side and sleeve seam
+- With *good sides together* pin the front to the back with along the sleeve and side seams, making sure the armhole seams line up.
```

Comment:
I think the "with along" bit is a typo. Tried to fix but please check whether it makes sense.
```suggestion
- With *good sides together* pin the front to the back with the sleeve and side seams matching, making sure the armhole seams line up across the front and back.
```

Reply:
mm let me fix this